src/cata_tiles.cpp

```diff
@@ -2500,7 +2498,7 @@ bool cata_tiles::draw_vpart( const tripoint &p, lit_level ll, int &height_3d )
     bool draw_highlight = cargopart && !veh->get_items( cargopart->part_index() ).empty();
 
     if( !veh->forward_velocity() ) {
-        g->u.memorize_tile( p, vpid, subtile, veh_dir );
+        g->u.memorize_tile( g->m.getabs( p ), vpid, subtile, veh_dir );
```

Comment:
`memorize_tile` may be called multiple times per tile, but only the most recent value is actually used. I see that it's one of many issues of the previous implementation, but it would be great to fix it as well at some point.